volatility3/framework/symbols/linux/__init__.py

```diff
@@ -46,6 +46,7 @@ def __init__(self, *args, **kwargs) -> None:
         self.set_type_class("sock", extensions.sock)
         self.set_type_class("inet_sock", extensions.inet_sock)
         self.set_type_class("unix_sock", extensions.unix_sock)
+        self.set_type_class("net_device", extensions.net_device)
```

Comment:
Will this exist on all kernels, or should this be an optional `set_type` (as a few lines below)?

Reply:
I think it will exist - but it never hurts to double check! I'll look into it.

Reply:
I confirm `net_device` exists at least from 2.6.30, so it should be safe.